R/addins.R

```diff
@@ -0,0 +1,13 @@
+custom_constraint <- function() {
```

Comment:
so a user would use this? I don't quite see the use case

Reply:
Ideally, this would make it so that people can use the addin part of R to format their custom constraints (so that it gets the plan distr pieces right which would avoid questions in the future about how to do it. We can drop it if you think it isn't worth it.

Reply:
If we want to do addins then we should probably think about larger template-like stuff, e.g. a menu item to insert a whole workflow or something like that. To me this doesn't have any value since it just inserts the function signature with a totally empty function body.  The example at the bottom of the `add_constr_custom()` documentation I think suffices to illustrate the usage. We can add more examples there if need be

Reply:
Okay, let's drop this for now then. It might be worth thinking about larger templates after we're happy with this